tests/foreman/cli/test_host_collection.py

```diff
@@ -42,126 +39,91 @@
 
 
 class HostCollectionTestCase(CLITestCase):
-    """Host Collection CLI tests."""
-
-    org = None
-    new_cv = None
-    promoted_cv = None
-    new_lifecycle = None
-    library = None
-    default_cv = None
-
-    # pylint: disable=unexpected-keyword-arg
-    def setUp(self):
-        """Tests for Host Collections via Hammer CLI"""
-        super(HostCollectionTestCase, self).setUp()
-
-        if HostCollectionTestCase.org is None:
-            HostCollectionTestCase.org = make_org(cached=True)
-        if HostCollectionTestCase.new_lifecycle is None:
-            HostCollectionTestCase.new_lifecycle = make_lifecycle_environment(
-                {u'organization-id': HostCollectionTestCase.org['id']},
-                cached=True
-            )
-        if HostCollectionTestCase.library is None:
-            HostCollectionTestCase.library = LifecycleEnvironment.info({
-                u'organization-id': HostCollectionTestCase.org['id'],
-                u'name': ENVIRONMENT,
-            })
-        if HostCollectionTestCase.default_cv is None:
-            HostCollectionTestCase.default_cv = ContentView.info({
-                u'organization-id': HostCollectionTestCase.org['id'],
-                u'name': DEFAULT_CV
-            })
-        if HostCollectionTestCase.new_cv is None:
-            HostCollectionTestCase.new_cv = make_content_view(
-                {u'organization-id': HostCollectionTestCase.org['id']}
-            )
-            HostCollectionTestCase.promoted_cv = None
-            cv_id = HostCollectionTestCase.new_cv['id']
-            ContentView.publish({u'id': cv_id})
-            result = ContentView.version_list({u'content-view-id': cv_id})
-            version_id = result[0]['id']
-            ContentView.version_promote({
```

Comment:
I can't see you've used similar CVs/CVVs anywhere in tests. Were these lines absolutely useless and never used?

Reply:
Already found the answer. That's weird. Thanks for removing redundant things :)